internal/cmd/skupper/site/nonkube/site_create.go

```diff
@@ -77,29 +69,34 @@ func (cmd *CmdSiteCreate) ValidateInput(args []string) []error {
 			validationErrors = append(validationErrors, fmt.Errorf("site name is not valid: %s", err))
 		}
 		cmd.siteName = args[0]
-
 	}
 
-	if cmd.Flags.LinkAccessType != "" {
-		ok, err := linkAccessTypeValidator.Evaluate(cmd.Flags.LinkAccessType)
+	if cmd.Flags != nil && cmd.Flags.Output != "" {
+		ok, err := outputTypeValidator.Evaluate(cmd.Flags.Output)
 		if !ok {
-			validationErrors = append(validationErrors, fmt.Errorf("link access type is not valid: %s", err))
+			validationErrors = append(validationErrors, fmt.Errorf("output type is not valid: %s", err))
 		}
 	}
 
-	if !cmd.Flags.EnableLinkAccess && len(cmd.Flags.LinkAccessType) > 0 {
-		validationErrors = append(validationErrors, fmt.Errorf("for the site to work with this type of linkAccess, the --enable-link-access option must be set to true"))
+	if cmd.Flags != nil && cmd.Flags.BindHost == "" && cmd.Flags.EnableLinkAccess {
+		validationErrors = append(validationErrors, fmt.Errorf("bindhost should not be empty"))
```

Comment:
I am wondering why not set `0.0.0.0` as the default value for bind host?
It is currently required, but if enable-link-access is not true, it is not used for anything.

Reply:
defaulted bind-host